ext/shmop/shmop.stub.php

```diff
@@ -4,15 +4,15 @@
 
 final class Shmop {}
 
-function shmop_open(int $key, string $flags, int $mode, int $size): Shmop|false {}
+function shmop_open(int $key, string $mode, int $permissions, int $size): Shmop|false {}
 
-function shmop_read(Shmop $shmid, int $start, int $count): string {}
+function shmop_read(Shmop $shmop, int $offset, int $size): string {}
```

Comment:
```suggestion
function shmop_read(Shmop $shmop, int $offset, int $length): string {}
```
To match substr etc?

Reply:
I'd still prefer size here because e.g. `stream_set_read_buffer()` et al. uses `$size` too, and I think when we talk about a chunk of memory, "size" sounds better than "length". Does it make sense to you?

Reply:
fread() using $length, which seems like the most direct analogy here: https://github.com/php/php-src/blob/efe6006ced4f91d0df1fdb0aef7f0079273ac0d0/ext/standard/basic_functions.stub.php#L796

Reply:
I've just re-read the manual, and noticed that there is the `shmop_size()` function which is mentioned in connection with the 3rd parameter of `shmop_read()`: `The number of bytes to read. 0 reads shmop_size($shmid) - $start bytes`.

Also, `shmop_open()` describes the 3rd param as `The size of the shared memory block you wish to create in bytes`. And I'm wondering whether this sentence still makes sense if we swapped "size" to "length"? In my opinion, the former reads much better, but I'd be curious how you see that.